<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payview</class>
 <widget class="QDialog" name="Payview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>payview â€” The Dollar Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Payroll Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="area"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="outputEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportCsvBtn">
     <property name="text">
      <string>Export CSV</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
